curator-framework/src/main/java/org/apache/curator/framework/imps/CreateBuilderImpl.java

```diff
@@ -1224,7 +1216,7 @@ String adjustPath(String path) throws Exception
         if ( protectedMode.doProtected() )
         {
             ZKPaths.PathAndNode pathAndNode = ZKPaths.getPathAndNode(path);
-            String name = getProtectedPrefix(protectedMode.protectedId()) + pathAndNode.getNode();
+            String name = ProtectedUtils.getProtectedPrefix(protectedMode.protectedId()) + pathAndNode.getNode();
```

Comment:
Let's move this into `ProtectedUtils`. Something like `ProtectedUtils.toProtectedZNode(String znodeName)`

Reply:
Fine for me, I will move it there.
Public also? My opinion is same as before (protected), even better as default method of the interface